### PR TITLE
DSPDC-1005 Add MVP schema.

### PR DESCRIPTION
**Pull request by danxmoran**
Hard-coded for now to match the specification [here](https://docs.google.com/document/d/1NsibP8g-NeLnksxlcBWQsSj5Zg_uimCDaAF1qB_qkjg/edit).

### CODE REVIEW

File: schema/mvp.json [hunk 1]
```diff
@@ -0,0 +1,326 @@
...
+        { "name": "content", "datatype": "string" }
+      ],
+      "primaryKey": ["project_id"],
+      "partitionMode": "date",
+      "datePartitionOptions": {
+        "column": "version"
+      }
+    },
+    {
+      "name": "protocol",
```
Comment (danxmoran) [inline schema/mvp.json:201]: Unclear if this table is needed: https://github.com/HumanCellAtlas/metadata-schema/issues/1269#issuecomment-623534894

Comment (danxmoran) [reply schema/mvp.json:201]: Answer: we're still not sure, but having the extra table doesn't hurt anything

File: schema/mvp.json [hunk 2]
```diff
@@ -0,0 +1,326 @@
...
+        { "name": "content", "datatype": "string" }
+      ],
+      "primaryKey": ["organoid_id"],
+      "partitionMode": "date",
+      "datePartitionOptions": {
+        "column": "version"
+      }
+    },
+    {
+      "name": "specimen_from_json",
```
Comment (raaidbroad) [inline schema/mvp.json:69]: should this be "specimen_from_organism"?

File: schema/mvp.json [hunk 3]
```diff
@@ -0,0 +1,326 @@
...
+      "columns": [
+        { "name": "specimen_from_json_id", "datatype": "string" },
```
Comment (raaidbroad) [inline schema/mvp.json:71]: should this be "specimen_from_organism_id"?

File: schema/mvp.json [hunk 4]
```diff
@@ -0,0 +1,326 @@
...
+        { "name": "version", "datatype": "timestamp" },
+        { "name": "content", "datatype": "string" }
+      ],
+      "primaryKey": ["specimen_from_json_id"],
```
Comment (raaidbroad) [inline schema/mvp.json:75]: should this be "specimen_from_organism_id"?

**Pull request merged**